src/coreclr/jit/morph.cpp

```diff
@@ -9371,7 +8652,8 @@ GenTree* Compiler::fgMorphCall(GenTreeCall* call)
         // This is call to CORINFO_HELP_VIRTUAL_FUNC_PTR with ignored result.
         // Transform it into a null check.
 
-        GenTree* thisPtr = call->gtCallArgs->GetNode();
+        assert(call->gtArgs.CountArgs() >= 1);
+        GenTree* thisPtr = call->gtArgs.GetArgByIndex(0)->GetNode();
```

Comment:
Not your doing, but calling arg0 `this` seems a bit confusing...?
```suggestion
        GenTree* objPtr = call->gtArgs.GetArgByIndex(0)->GetNode();
```